api/prisma/seed-helpers/listing-factory.ts

```diff
@@ -95,16 +95,30 @@ export const listingFactory = async (
       : new Date(),
     listingMultiselectQuestions: optionalParams?.multiselectQuestions
       ? {
-          create: optionalParams.multiselectQuestions.map(
-            (question, index) => ({
-              multiselectQuestions: {
-                connect: {
-                  id: question.id,
+          create: [
```

Comment:
We weren't correctly seeding preferences and programs - the ordinals were getting added as if they were all the same type of multiselect question, so a program was never appearing because it was ordinal 3 in the data but needed to be ordinal 1